protocol/messenger_config.go

```diff
@@ -275,6 +275,20 @@ func WithPushNotifications() func(c *config) error {
 	}
 }
 
+func WithBackupDisabled() func(c *config) error {
```

Comment:
The same for this and the one below, where do we use them?

Reply:
It is used in the bridge repo: https://github.com/status-im/matterbridge/pull/1/files#diff-a04d7ca2cd2cc8163c50e09baf54a0c66a04242f0ced84601144cb2a7abd87c1R313

I know that we have settings switch, but bridge's settings table is empty. And default setting (for NoRows error) is true. Maybe we can change it to false? Then I can remove this one.
Also, what about checking BackupEnabled flag before running a goroutine in startBackupLoop() ?

Reply:
> what about checking BackupEnabled flag before running a goroutine in startBackupLoop()?

I am pretty sure, that you can do that, actually that's how it should be.

> Maybe we can change it to false?

I don't think we need it at all.

When you call `backend.StartNode` you can pass `false` for `ProcessBackedupMessages` via `NodeConfig` and you can also check how to set the initial setting via the passed node config which will set `BackupEnabled` in the settings table (check the usage of `CreateSettings` function).

Reply:
@saledjenic I fixed naming according to our discussion